python/lsst/ts/linearstage/config_schema.py

```diff
@@ -2,103 +2,62 @@
 
 CONFIG_SCHEMA = yaml.safe_load(
     """
-$schema: http://json-schema.org/draft-07/schema#
+$schema: http://json-schema.org/2020-12/schema#
 $id: https://github.com/lsst-ts/ts_LinearStage/blob/master/schema/LinearStage.yaml
 # title must end with one or more spaces followed by the schema version, which must begin with "v"
-title: LinearStage v3
+title: LinearStage v4
 description: Schema for LinearStage configuration files
 type: object
-additionalProperties: false
 properties:
-    stage_type:
-        description: Type of stage being controlled.
-        type: string
-        enum: [Zaber, Igus]
-    target_position_minimum:
-        description: >
-          Minimum target position in mm
-        type: number
-    target_position_maximum:
-        description: >
-          Maximum target position in mm
-        type: number
-    serial_port:
-        description: USB port for the serial interface
+  linear_stage_config:
```

Comment:
Please name this field `instances`. That matches reality very well: it is the configuration for one instance  (one specific SAL index) of your CSC and is what other similar CSCs do.